merkle-tree/bounded-vec/src/lib.rs

```diff
@@ -809,6 +810,24 @@ mod test {
         bounded_vec
     }
 
+    #[test]
+    fn test_bounded_vec_metadata() {
```

Comment:
This test tests metadata parsing/serialization right?
(It would be easier if the title reflects this.)